docs/src/flags.md

```diff
@@ -10,20 +10,20 @@ Several flag variables are also defined, each with multiple entries defined.
 Here is an example of use of `CapControlModes.KVAR`:
 
 ```julia
-CapControls.Mode(CapControlModes.KVAR)
+CapControls.Mode(Lib.CapControlModes_KVAR)
```

Comment:
Also need `CapControlModes_KVAR` in the line above.

Reply:
Resolved in 002a580.